[Security] Multiple OpenTelemetry packages have a known moderate severity vulnerability

C:\dev\OtelPlayground-Validation\src\SpaceStationMonitor\SpaceStationMonitor.csproj : warning NU1902: Package 'OpenTelemetry.Api' 1.15.1 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-g94r-2vxg-569j

C:\dev\OtelPlayground-Validation\src\SpaceStationMonitor\SpaceStationMonitor.csproj : warning NU1902: Package 'OpenTelemetry.Exporter.OpenTelemetryProtocol' 1.15.1 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-mr8r-92fq-pj8p

C:\dev\OtelPlayground-Validation\src\SpaceStationMonitor\SpaceStationMonitor.csproj : warning NU1902: Package 'OpenTelemetry.Exporter.OpenTelemetryProtocol' 1.15.1 has a known moderate severity vulnerability, https://github.com/advisories/GHSA-q834-8qmm-v933

--- a/src/SpaceStationMonitor/SpaceStationMonitor.csproj
+++ b/src/SpaceStationMonitor/SpaceStationMonitor.csproj
@@ -8,9 +8,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry" Version="1.15.1" />
-    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.15.1" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.15.1" />
+    <PackageReference Include="OpenTelemetry" Version="1.15.3" />
+    <PackageReference Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.15.3" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.15.3" />
   </ItemGroup>
 
 </Project>

